frontend/contact/index.html

```diff
@@ -125,6 +123,21 @@ <h3>My domain (or one of its subdomains) is broken due to preloading, and waitin
 
   <section id="other">
     <h2><a class="hash-link" href="#other">Other questions</a></h2>
+
+    <h3>Customer support for my service provider told me to contact you for support</h3>
+    <p>
+      We apologize that your service provider instructed you to contact us, as we are unable to assist you.
+    </p>
+    <p>
+      If you are attempting to remove your domain from the HSTS preload list, use the form at <a href="/removal">hstspreload.org/removal</a> and see other questions in the <a href="#removal">removal</a> section of this FAQ. Your service provider's customer support department should be able to provide all needed support to help you remove your domain from the HSTS preload list.
+    </p>
+
+    <h3>I have an HSTS question that is not related to HSTS preloading</h3>
+    <p>
+      We do not provide technical support or answer questions about anything not directly related to HSTS preloading. You will need to find support for your HSTS question elsewhere.
+    </p>
+
+    <h3>My question isn't answered on this page</h3>
     <p>
       If you have other questions that aren't covered by this site, you can <a href="mailto:hstspreload@chromium.org">send us an email</a>. We will not respond to emails asking questions that are answered on this site.
```

Comment:
This answer feels a bit at odds with your new Q&As above. I do think both are worth saying -- we can't help individual domains troubleshoot their HSTS preloading, and also please reach out to us if you have general preloading questions that aren't addressed here or want to preload a large domain and/or a TLD. My meta comment is that we should work to make that distinction clearer, although that's hard.

Reply:
I tweaked the opening of this sentence.